### ISSUE TITLE
Various UX things

### ISSUE THREAD

**Issue by cloudhead** [radicle-dev/radicle-interface#624]
* [ ] I think we should show the full DID here:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/40774/222748704-e79d6cf7-a861-4b92-ad81-9af54bbe28b6.png">
* [ ] Clicking on the project name when on the project page, eg. `heartwood` goes to a URL with `../<rid>/tree` at the end, it should just go to `../<rid>`, we could also fix this by having `/tree` render but not be an actual route when there's no commit-id after it.
* [ ] Project URL on seed should allow trailing slash, eg. `../<rid>/`

**Comment by rudolfs** [comment]
Is this what you had in mind?

<img width="519" alt="Screenshot 2023-03-03 at 17 40 31" src="https://user-images.githubusercontent.com/158411/222777000-bda32af5-6a8e-4cc4-90c1-8f6ee0e2eb15.png">

**Comment by cloudhead** [comment]
Yeah that's right, I think we should use a grey that is one notch brighter though.

For the seed address, the `did:key:` prefix should not be present, only the key should be used. I also think we should maybe omit the port, as that's the default port :)

**Comment by rudolfs** [comment]
> Ah and I think we can use the full NodeID as well, there is enough horizontal space there.

I think it looks a bit off when it's the full length. Wdyt?

<img width="1840" alt="Screenshot 2023-03-06 at 10 12 03" src="https://user-images.githubusercontent.com/158411/223066492-3ddf54aa-97c4-4b0c-a5e1-cd72fa89edd0.png">

**Comment by rudolfs** [comment]
Bumped the gray one level up:

<img width="521" alt="Screenshot 2023-03-06 at 10 14 39" src="https://user-images.githubusercontent.com/158411/223067094-33734aa8-2088-42d4-851d-5e94ca333792.png">

The blue we can't bump because the next level is actually lighter:
<img width="264" alt="Screenshot 2023-03-06 at 10 14 48" src="https://user-images.githubusercontent.com/158411/223067162-7f5f9502-21fe-44e2-973d-a267902a29a4.png">

**Comment by rudolfs** [comment]
Turns out the loaders are still there and work, it's just that some of the requests are too fast for the loader to even render. If I add a 1sec delay to the api requests then the loaders show up properly:

https://user-images.githubusercontent.com/158411/223078673-692b5abf-47a9-4b1d-bc4b-500efed81e67.mov

**Comment by cloudhead** [comment]
> Turns out the loaders are still there and work, it's just that some of the requests are too fast for the loader to even render. If I add a 1sec delay to the api requests then the loaders show up properly:

Yes, but that's a problem... we don't want the folder opening without a loader and without files. So either we remove the delay, or we don't open the folder until the files are loaded?

**Comment by cloudhead** [comment]
> Bumped the gray one level up:
> The blue we can't bump because the next level is actually lighter:

Great, the dark blue is fine for me.

**Comment by cloudhead** [comment]
> I think it looks a bit off when it's the full length.
Agreed. Either we have to use a smaller font, or we go back to the trimmed version.. I'd say let's use the trimmed verison for now, all one color, and I'll play around with it at some point.

**Comment by rudolfs** [comment]
> Yes, but that's a problem... we don't want the folder opening without a loader and without files. So either we remove the delay, or we don't open the folder until the files are loaded?

Looked at it again. I think the issue is that we add a 200ms delay to the loader to prevent it from flickering when the request is fast. So there's a tradeoff.

We can either:
  - lower the delay
  - remove the delay and be okay with the loader flickering on fast requests
  - keep it as is, slow requests still show the loader, fast requests don't flicker

This is when the delay is removed entirely:
https://user-images.githubusercontent.com/158411/223386543-adacff7a-8937-4634-9eae-4f0743cb58d7.mov

**Comment by cloudhead** [comment]
For now I would probably remove the delay completely, so that there's only two states: *loading*, and *loaded*, regardless of how long it takes.

Eventually though, I think having it work like the files, where the folder doesn't open until the contents are loaded, but a small single loader is shown to the side, would make most sense.
